tests/src/Kernel/DataProducer/XML/XMLXpathTest.php

```diff
@@ -12,7 +12,7 @@ class XMLXpathTest extends XMLTestBase {
   /**
    * @covers \Drupal\graphql\Plugin\GraphQL\DataProducer\XML\XMLXpath::resolve
    */
-  public function testXMLXpath() {
+  public function testXmlxPath() {
```

Comment:
As words describing the function are basically `test XML XPath`, let's camelCase by the words, which for this case results into `testXmlXpath`

Reply:
Pushed new version